src/noahowpLogger.f90

```diff
@@ -19,6 +19,12 @@ module noahowp_log_module
    logical :: opened_once = .false.
    logical :: logger_initialized = .false.   ! Flag to track if the logger has been initialized
 
+   ! To use Logger while running this module stand-alone, set
+   ! the following environment variables, case-sensitive:
+   !     NGEN_EWTS_LOGGING=ENABLED
+   !     NGEN_LOG_FILE_PATH=<full pathname for log file>
+   !     NOAHOWP_LOGLEVEL=<DEBUG, INFO, WARNING, SEVERE, FATAL> 
+   !
```

Comment:
I suggest adding this information to the INSTALL.md file under the 'Run' section.